packages/emails/templates/_base-email.ts

```diff
@@ -48,6 +48,17 @@ export default class BaseEmail {
     const payload = this.getNodeMailerPayload();
     const parseSubject = z.string().safeParse(payload?.subject);
     const payloadWithUnEscapedSubject = {
+      headers: {
```

Comment:
This implementation is specific to SendGrid, right?

If so, we should try to move this functionality to some sort of higher level function/layer that is specific to SendGrid and isn’t in the BaseEmail